### PR TITLE
Read configuration from configmaps in user namespaces

### PR DESCRIPTION
**Pull request by dkondyrev**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,14 +1,43 @@
...
 
 
+def read_config_files(configuration):
+    """
+    Read configuration files from configmaps in all available namespaces
+    :param configuration: Kubernetes client configuration
+    :return: list of user elastalert rules
+    """
+    api_instance = kubernetes.client.CoreV1Api(kubernetes.client.ApiClient(configuration))
+    configmap_list = api_instance.list_config_map_for_all_namespaces()
```
Comment (rvadim) [inline main.py]: Administrator will restrict for reading all configmaps from any namespaces. So we should write exact query.

File: main.py [hunk 2]
```diff
@@ -1,14 +1,43 @@
...
+
+    user_configs = []
+    for configmap in configmap_list:
+        if configmap.metadata.name == 'elastalert-rules':
+            for rule_name in configmap.data:
+                user_rule = yaml.safe_load(configmap.data[rule_name])
```
Comment (rvadim) [inline main.py:29]: We need handle exception here, due to it ruin whole loop.

**Review by rvadim — CHANGES_REQUESTED** [review]
Good. Some minor fixes required.

File: main.py (hunk 1 above)
Comment (dkondyrev) [reply main.py]: Changed
```suggestion
    configmap_list = api_instance.list_config_map_for_all_namespaces(field_selector='metadata.name=elastalert-rules')
```

File: main.py (hunk 2 above)
Comment (dkondyrev) [reply main.py:29]: Fixed

**Review by rvadim — APPROVED** [review]
LGTM

**Review by geshido — APPROVED** [review]

**Pull request merged**